sopel/loader.py

```diff
@@ -104,10 +104,35 @@ def clean_callable(func, config):
 
 
 def is_triggerable(obj):
-    return any(
-        hasattr(obj, attr)
-        for attr in ('rule', 'event', 'intents', 'commands',
-                     'nickname_commands'))
+    """Check if ``obj`` can handle the bot's triggers.
+
+    :param obj: any :term:`function` to check.
+    :return: ``True`` if ``obj`` can handle the bot's triggers.
```

Comment:
I _think_ we've been omitting trailing `.` on param/return lines.

Reply:
```suggestion
    :return: ``True`` if ``obj`` can handle the bot's triggers
```